### ISSUE TITLE
Battery full-at sometimes displays zero

### ISSUE THREAD

**Issue by SamHH** [Alexays/Waybar#666]
Using the new [full-at](https://github.com/Alexays/Waybar/wiki/Module:-Battery#config) option for the battery module in 0.9.2, it sometimes displays the correct number, and sometimes displays zero. I haven't nailed down why this is yet, I'll pop back with a more helpful repro if I figure it out. In the meantime, here's the relevant part of my config:

```
"battery": {
    "full-at": 72,
    "states": {
        "warning": 30,
        "critical": 15
    },
    "format": "{capacity}% {icon}",
    "format-icons": ["", "", "", "", ""]
},
```

The bug only occurs when the `full-at` option is specified, and the formatting also displays as if the battery is dead (i.e. it triggers the critical threshold).

**Comment by Alexays** [comment]
An error should be printed can you run waybar in the terminal and print the output here?

**Comment by SamHH** [comment]
It's happening right now, running `waybar` spawns a second bar with the same issue but unfortunately no relevant errors are printed (the width one is always printed):

```
[2020-04-15 11:52:19.910] [info] Resources files: /home/<user>/.config/waybar/config, /home/<user>/.config/waybar/style.css
[2020-04-15 11:52:20.023] [warning] Requested width: 0 exceeds the minimum width: 1200 required by the modules
```

With `-l trace`:

```
$ waybar -l trace                                                                                                                                                                                                                                                                        ~
[2020-04-15 11:55:00.182] [info] Resources files: /home/<user>/.config/waybar/config, /home/<user>/.config/waybar/style.css
Unknown option -l
[2020-04-15 11:55:00.186] [debug] Output detected: eDP-1 (Apple Computer Inc Color LCD)
[2020-04-15 11:55:00.302] [warning] Requested width: 0 exceeds the minimum width: 1200 required by the modules
[2020-04-15 11:55:00.302] [debug] Set surface size 1200x45 for output eDP-1
[2020-04-15 11:55:00.302] [debug] Set exclusive zone 45 for output eDP-1
```

**Comment by SamHH** [comment]
Any chance of a point release with this included? Or is it full steam ahead to 1.0 now?

When I undock I'm resorting to catting /sys haha. And yesterday I forgot to plug in and the laptop of course died. Perma 0% is a challenge :stuck_out_tongue:
